spec/features/staff/users_can_view_an_organisation_as_xml_spec.rb

```diff
@@ -8,7 +8,7 @@
     context "when the user is viewing the BEIS organisation show page" do
       scenario "they cannot download the organisation's projects as XML" do
         beis = user.organisation
-        _project = create(:project_activity, organisation: beis)
+        _project = create(:programme_activity, organisation: beis)
```

Comment:
This changed from `create(:project_activity)` to `create(:programme_activity)`, so the variable should be updated accordingly.